lib/Flow/Operation.php

```diff
@@ -116,7 +117,17 @@ public function onEvent(string $eventName, Event $event, IRuleMatcher $ruleMatch
 				}
 
 				$room = $this->getRoom($token, $uid);
+				if (!($room->getReadOnly() !== Room::READ_WRITE)) {
```

Comment:
```suggestion
				if ($room->getReadOnly() === Room::READ_WRITE) {
```

Reply:
yeah, it was a bug before. Didn't mean to invert it. read-write is okay, read-only is what we want to filter out.